File: Modul 3/Source Code/Python - Modul 3/PRAK304-2410817310008-MAnshary.py
bil = int(input("Masukkan Bilangan: "))

if bil >=1 and bil<=9:
    print("Satuan")
elif bil==10 or bil>=20 and bil<=99:
    print("Puluhan")
elif bil>=11 and bil<=19:
    print("Belasan")
elif bil==0:
    print("Nol")
else:
    print("Anda Menginput Melebihi Limit Bilangan")

File: Modul 3/Source Code/Modul 3/PRAK304-2410817310008-MAnshary.c
#include<stdio.h>
int main(){
    int bil;

    printf("Masukkan Nilai Anda: ");
    scanf("%d", &bil);
if(bil>=1 && bil<=9){
    printf("Satuan");
}
else if(bil==10 || bil>=20 && bil<=99){
    printf("Puluhan");
}
else if(bil>=11 && bil<=19){
    printf("Belasan");
}
else if(bil==0){
    printf("Nol");
}
else{
    printf("Anda Menginput Melebihi Limit Bilangan");
}
}